hendrix/deploy/base.py

```diff
@@ -24,43 +26,38 @@ class HendrixDeploy(object):
     the HendrixService on a single or multiple processes.
     """
 
-    def __init__(self, action='start', options={},
-                 reactor=reactor, threadpool=None):
-        self.action = action
-        self.options = hx_options()
-        self.options.update(options)
-        self.services = []
-        self.resources = []
```

Comment:
I like that this crufty section has been removed, but resources is an interesting and perhaps special case:

* We do need resources in order to addHendrix
* We do not have resources by default if we aren't using settings.

* Thus, if the user doesn't specify resources, we do need this empty list.